Formula/swctl.rb

```diff
@@ -0,0 +1,21 @@
+class Swctl < Formula
+  desc "SiteWhere Control CLI"
+  homepage "https://sitewhere.io/"
+  url "https://github.com/sitewhere/swctl.git",
+      tag:      "v0.4.2",
+      revision: "22227b0c66bf5981a775f4cd7eb91676de53472d"
+  license "Apache-2.0"
+  head "https://github.com/sitewhere/swctl.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install "bin/swctl"
+  end
+
+  test do
+    system "#{bin}/swctl", "version"
+    assert_predicate testpath/"bin/swctl", :exist?
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.